addons/godot-next/nodes/callback_delegator.gd

```diff
@@ -52,9 +52,13 @@ func _get_property_list() -> Array:
 	return [ PropertyInfo.new_resource("_elements").to_dict() ]
 
 func _ready() -> void:
-	_handle_notification("_ready")
+	if not Engine.editor_hint:
+		_handle_notification("_ready")
 
 func _enter_tree() -> void:
+	if Engine.editor_hint:
```

Comment:
Can't this also use `_handle_notification` then? Since it's also exiting early if in the editor?